### DISCUSSION TITLE
Formatter breaks before `?` when wrapping long lines with optional chaining?

### DISCUSSION THREAD

**Discussion by MrTact** [rome/tools#4783] [General]
Working on running `rome format` against my codebase, and I noticed that with a long line that has optional chaining in it, when rome breaks at a chained property/function call, it sometimes (always?) breaks the chain before the `?`, which has the next line starting with `?.` Something like this, for example:

```typescript
    const foo = this?.is?.a?.really?.long?.chain
        ?.of?.optional?.calls();
```

Wondering whether this is by design. Just struck me as an odd choice, but I wasn't able to find other discussion/issues mentioning it.

Thanks!

**Comment by Conaclos** [comment]
Rome mimics the behavior of Prettier in this context.
